python/lsst/gdb/afw/printers.py

```diff
@@ -752,7 +796,7 @@ def to_string(self):
                                                          self.val["_numIter"],
                                                          self.val["_andMask"])
 
-    #-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+    # -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
```

Comment:
I don't think anyone would object if we removed lines like this. They are no longer part of the coding standard so I'd prefer they disappeared in this older code.